scripts/sanitycheck

```diff
@@ -2369,6 +2369,7 @@ class TestSuite:
 
         return self.goals
 
+    # --save-tests
```

Comment:
What is this comment?

Reply:
^, fine with me otherwise (though I'm not a `sanitycheck` expert).

Reply:
This comment is just header showing where the `--save-tests` implementation is. It took me a long time to find it (without the header) because the function is called `def run_report()` for a reason unknown to me.

Reply:
Would it make sense to have a comment a bit more descriptive?

Reply:
I just changed the name of the function instead, it was used once and only once.